src/drivers/intel/hda/hda-dma.c

```diff
@@ -67,8 +67,8 @@ DECLARE_TR_CTX(hdma_tr, SOF_UUID(hda_dma_uuid), LOG_LEVEL_INFO);
 #define HDA_STATE_RELEASE	BIT(0)
 
```

Comment:
@juimonen wait a second. The manual says the alignment should be either 32/64 bits or bytes? We need to clarify this in the commit message.

Because now we are chaning alignment from 0x20 to 0x8 and it doesn't makes sense according to the commit message.

Reply:
yeah I want to change it from 32 bytes to 8 bytes (64 bits), so the "bigger" minimum from 64 bits.

Reply:
So in dw_dma we have it set to 4 bytes (32 bits), not sure why it is 32 bytes in hda dma... I read the spec in bits, but someone else from the team can chime in. And I want to see what happens with CI...

Reply:
yeah, this might be just wrong "fix", so something is fishy in the host copy if the size of 1 pipeline processing period happens to be under dma alignment size... I think they should not depend on each other.